cmd/influxd/launcher/cmd.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/influxdata/influxdb/v2/storage"
 	"github.com/influxdata/influxdb/v2/v1/coordinator"
 	"github.com/influxdata/influxdb/v2/vault"
+	natsserver "github.com/nats-io/gnatsd/server"
```

Comment:
heads up that this is still v1 of the server for v2, this would be needed:
```
"github.com/nats-io/nats-server/v2/server"
```
(it is backwards compatible as well)